INSTALL.md

```diff
@@ -337,10 +337,14 @@ sudo apt-get update
 sudo apt-get -y install bison build-essential cmake flex git libedit-dev \
   libllvm6.0 llvm-6.0-dev libclang-6.0-dev python zlib1g-dev libelf-dev libfl-dev python3-distutils
 
-# For Eoan (19.10) or Focal (20.04.1 LTS)
+# For Eoan (19.10)
```

Comment:
Can you delete this whole `19.10` section? Or adjust the `clang`/`llvm` libs installed here? `llvm11` is now the minimum version

Reply:
done. have deleted.